cypress/support/commands.ts

```diff
@@ -44,7 +44,9 @@ Cypress.Commands.add("addMachine", (hostname = generateName()) => {
   cy.get("input[name='hostname']").type(hostname);
   cy.get("input[name='pxe_mac']").type(generateMac());
   cy.get("select[name='power_type']").select("manual").blur();
-  cy.get("button[type='submit']").click();
+  cy.get("button[type='submit']", { timeout: LONG_TIMEOUT })
+    .should("be.enabled")
+    .click();
```

Comment:
As explained in the acceptance criteria, our goal here is to:

> Wait for ephemeral OS to be available or provide a workaround allowing the test to pass when it’s unavailable

Generic timeout wait specific enough. I'd recommend to talk to someone from the MAAS core team, share a screenshot of the failing test where ephemeral message is visible and find if there are better ways of detecting this.

Reply:
Okay, thanks I reach out to someone concerning this, this is not up for review yet, I created a draft PR as a sandbox to get Cypress test results.